### PR TITLE
Update README.md - Added another bullet point

### PR DESCRIPTION
**Pull request by addyapproved**
Please review

### CODE REVIEW

**Review by Adomonir — CHANGES_REQUESTED** [review]
You should add more details

**Review by Adomonir — CHANGES_REQUESTED** [review]
Add more features to line item

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - Facilitator: @addyapproved
 - 16 participants
 - Generate Scripts
+- Remove Repository2
   
 ## Class Resources
```
Comment (Adomonir) [inline README.md:10]: Test

**Review by addyapproved — COMMENTED** [review]
Yes

**Pull request merged**